2_adventures/001/alexsimmons1964/05/terminal_commands.md

```diff
@@ -0,0 +1,77 @@
+Terminal commands I'm using with Windows Powershell terminal
+
+Thought I'd list the terminal commands I've used as a way to emphasise my learning.
+
+cd
+
+change directory/folder - and some variants of it's usage:
+
+cd ..
+
+to go back up one level
+
+cd ~
+
+to go back to my C:\Users\Alex directory/folder on my Windows PC
+
+cd directoryname1/directoryname2 
+
+etc to navigate to a specific sub directory from where I am currently located
+
+I'm slow at this navigation business.
+
+
+ls
+
+list out whatever is in the directory
+files, folders, anything.
+using this to help me see what's in the next level down I need to navigate to.
+
+
+mkdir directoryname
+
+creating a new directory/folder with the name "directoryname"
+so far I am only game to first navigate with cd commands to the directory in which I'd 
+like to add a new directory, then add a new folder with mkdir. 
+OK so I'm a pussy, or an octocat, but baby steps right?
+
+
+git branch
+
+shows me the various branches that exist in my "repository". Not sure if I'm using the 
+term "repository" correctly.
+for now I'm just going to stay in master branch and add/make changes to that. I figure 
+that at the moment all I'm doing is adding text files, I can't really do much harm.
+I'm not sufficiently comfortable with the relationships between what's on my PC and 
+what's on my github to use branches just yet. I'm liable to get lost.
+
+
+git status
+
+tells me if I have made any changes since last "commit". Suggests perhaps I need to do a...
+
+
+git add -A
+
+which as I understand it place all changes in a "staging area", ready to be committed to 
+the branch I am currently in/on/at/loitering about.
+The "-A" is probably a bit over the top and no doubt I'll need to learn to be more subtle 
+with my use of git add commands.
```

Comment:
You will.  But for now without confidential information in your repository to avoid exposing to the public eyes, you are safe.